src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,100 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLIER = 2;
+
+    private Node<K, V>[] buckets;
+    private int threshold;
+    private int size;
+
+    public MyHashMap() {
+        this.buckets = new Node[DEFAULT_CAPACITY];
+        this.threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
```

Comment:
Instead of calculating getIndex(key) each time, you can just create int bucketIndex = getIndex(key); and use it instead

Reply:
done